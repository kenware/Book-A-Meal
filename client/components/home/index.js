import React, { Component, PropTypes } from 'react';
import { BrowserRouter, Route, Switch, Redirect, Link } from 'react-router-dom';
import { connect } from 'react-redux';
import { bindActionCreators } from 'redux';

import Header from '../header/index';
import './index.scss';
import * as actions from '../../redux/Action/action';


class Home extends Component {
  constructor(props) {
    super(props);
    this.state = {
    };
    this.onChange = this.onChange.bind(this);
  }

  componentWillReceiveProps(newProps) {

  }

  onChange(e) {
    const newLocal = this;
    const { state } = newLocal.state;
    state[e.target.name] = e.target.value;
    this.setState(state);
  }

  render() {
    return (
      <div>
        <div className="carousel-header">
          <div className="header-overlay" />
          <div className="caption">
            <h3 className="lead">Book A Meal Today</h3>
            <p className="lead">Health is wealth!</p>
          </div>
        </div>
        <Header />
      </div>

    );
  }
}

function mapStateToProps(state, ownProps) {
  return {
    message: 'state.message',
  };
}
function mapDispatchToProps(dispatch) {
  return { actions: bindActionCreators(actions, dispatch) };
}
export default connect(mapStateToProps, mapDispatchToProps)(Home);
