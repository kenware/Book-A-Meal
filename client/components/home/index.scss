body {
  margin: 0;
  font-family: Arial, Helvetica, sans-serif;
 
}
a:hover{
  color:lightseagreen;
}

select option{
  margin-left: 50px;
}
.li-style{
  list-style: none;
}
a{
  text-decoration: none;
}
h2{
  text-align: center;
  color:rgba(216, 186, 19, 0.795);
}
button{
  background-color:rgba(216, 186, 19, 0.795);
  color: black;
  
}
button:hover{
  background-color:rgb(81, 131, 114);
  color: rgb(195, 209, 218);
}
.carousel-header{
    margin-top: 4rem;
    background: url('../../image/carousel-1.jpg') center center fixed;
    -webkit-transition: opacity 1s;
    -moz-transition: opacity 1s;
    -ms-transition: opacity 1s;
    -o-transition: opacity 1s;
    transition: opacity 1s;
    width: 100%;
    height: 70vh;
    -webkit-background-size: cover;
    -moz-background-size: cover;
    -o-background-size: cover;
    background-size: cover;
    .caption{
      position: relative;
      top: 30vh;
      z-index: 10;
      color:white;
      text-align: center;
      font-size: 50px;
    }
    .header-overlay {
      background-color: #000;
      height: 70vh;
      left: 0;
      opacity: .5;
      position: absolute;
      right: 0;
      top: 4rem;
      z-index: 10;
    }
  }
  .circle{
    width: 100px;
    height: 100px;
    border-radius: 50%;
    background-color: rgba(216, 186, 19, 0.795);
 }
  @media(min-width:900px){
    .select-meal-day{
      display: flex;
      justify-content: space-between;
    }
  }
  .all-meal-step{
    display: flex;
    justify-content: space-between;
    flex-wrap: wrap;
    align-items: center;
    .meal-day{
      flex:1;
      display: flex;
      padding: 1rem 1rem 1rem 1rem;
    }
  }
  .p-color{
    color:rgb(5, 79, 145);
  } 
  .rounded-button{
    border-radius: 30%;
    height: 40px;
  }
  .rounded-circle{
    border-radius: 50%;
  }

  #App-logo {
    animation: App-logo-spin infinite 20s linear;
   
  }
  
  @keyframes App-logo-spin {
    from { transform: rotate(0deg); }
    to { transform: rotate(360deg); }
  }
  .meal-menu-row{
    display:flex;
    justify-content: space-around;
    flex-wrap: wrap;
    align-items: center;
    .meal-card-img{
      width:100%;
      height: 270px;
    }
    .meal-img-wrap {
      height: 270px;
    }
    .meal-img-wrap:hover {
      .meal-overlay {
        background-color: #000;
        width: 270px;
        height: 280px;
        opacity: .3;
        position: absolute;
        z-index: 10;
      }
      .meal-card-img{
        width:100%;
        height: 280px;
      }
     
    }
    .meal-menu-card {
      margin-top: 30px;
      width: 270px;
      height: 400px;
      flex:1;
      /* Add shadows to create the "card" effect */
      box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
      transition: 0.3s;
      
      .container {
        padding: 2px 16px;
      }
    }
    .meal-menu-flip {
      margin-top: 30px;
      width: 270px;
      height: 280px;
      flex:1;
      /* Add shadows to create the "card" effect */
      box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
      transition: 0.3s;
    }
    .meal-menu {
      padding: 4px 8px 4px 8px;
    }
 }
 
 
 
 .parallex{
  margin-top: 30px;
 display:flex;
 justify-content: space-around;
 flex-wrap: wrap;
 align-items: center;

 background: url("../../image/background6.jpg") center center fixed;
 -webkit-transition: opacity 1s;
 -moz-transition: opacity 1s;
 -ms-transition: opacity 1s;
 -o-transition: opacity 1s;
 transition: opacity 1s;
 -webkit-background-size: cover;
 -moz-background-size: cover;
 -o-background-size: cover;
 background-size: cover;
 .parallex-menu {
  margin-top: 2rem;
  margin-left: 1rem;
 }
 .parallex-card{
  width:400px;
  flex:1;
  padding-bottom: 2rem;
  background-color: #149dcc;
  background: linear-gradient(#149dcc, transparent);
  .container p{
    text-align: justify;
    padding: 0px 30px 0px 30px;
    color:white;
  }
}
.parallex-img{
  margin-top: 20px;
  border-radius: 0;
  padding: 2rem 2rem 0 2rem;
  background-color: #149dcc;
  
}
.parallex-footer{
  display:flex;
  justify-content: space-around;
  .footer-btn {
    flex:1;
    background-color:yellow;
  }
}
}



@media screen and (max-width: 600px) {
.parallex-card{
  width:200px;
  flex:1;
  padding-bottom: 2rem;
  background-color: #149dcc;
  background: linear-gradient(#149dcc, transparent);
}
}
.white-color{
  color: white;
}
.text-center{
  text-align: center;
}
.user-testimonial{
  background: url("../../image/testimonial.jpg") no-repeat center center fixed;
    -webkit-transition: opacity 1s;
    -moz-transition: opacity 1s;
    -ms-transition: opacity 1s;
    -o-transition: opacity 1s;
    transition: opacity 1s;
    -webkit-background-size: cover;
    -moz-background-size: cover;
    -o-background-size: cover;
    background-size: cover;
   
  }
  .testimonial-wrap{
    display: flex;
    justify-content: space-around;
    flex-wrap: wrap;
    align-items: center;
  }
  
  .user-testimonial-menu{
    width: 700px;
    flex:1;
    height: 20rem;
    margin-top: 5rem;
    background-color: #efefef;
    background: linear-gradient(#efefef, transparent);
    
  }
  @media screen and (max-width: 600px) {
  .user-testimonial-menu{
    width: 300px;
    flex:1;
    height: 20rem;
    margin-top: 5rem;
    background-color: #efefef;
    background: linear-gradient(#efefef, transparent);
    
  }
  }
  @media screen and (min-width: 600px) and (max-width: 900px) {
    .user-testimonial-menu{
      width: 400px;
      flex:1;
      height: 20rem;
      margin-top: 5rem;
      background-color: #efefef;
      background: linear-gradient(#efefef, transparent);
      
    }
    }
  .justify{
    text-align: justify;
  }
  .text-center {
    text-align: center;
  }
    .testimonial-photo{
      margin: -4rem 0px 0px 40%;
      display: inline-block;
      width: 120px;
      height: 120px;
      margin-bottom: 1.6rem;
      overflow: hidden;
      
    }
    .text-pad {
      margin: 0px 30px 0px 30px;
    }
    .l-r-pad-text{
      margin: 0px 20px 0px 20px;
    }
   .m-text{
    font-size: 20px;
  }
  .top-bot-margin {
    margin-top: 3rem;
    margin-bottom: 2rem;
  }
  .active-text-header{
  font-family: 'Tangerine', serif;
  }
