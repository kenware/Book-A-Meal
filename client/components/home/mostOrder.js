import React from 'react';
import { Link } from 'react-router-dom';
import PropTypes from 'prop-types';
/**
 * @param  {} {mostOrder} load most ordered menu
 */
export const mostOrdered = ({ mostOrder }) => (
  <div className="meal-menu-row">
    { mostOrder.map(meal =>
    (
      <div className="meal-menu" key={meal.id}>
        <div className="meal-menu-card">
          <div className="meal-img-wrap">
            <div className="meal-overlay" />
            <img src={meal.Meal.image} alt="Avatar" className="meal-card-img" />
          </div>
          <div className="container l-r-pad-text">
            <h4 className="p-color"><b>${meal.Meal.price}</b></h4>
            <p>{meal.Meal.name}</p>
            <Link to="/dashboard">View</Link>
          </div>
        </div>
      </div>
    ))
    }
  </div>
);
mostOrdered.propTypes = {
  mostOrder: PropTypes.array.isRequired,
};
export default mostOrdered;

