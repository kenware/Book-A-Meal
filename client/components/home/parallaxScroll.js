import React from 'react';
import { Link } from 'react-router-dom';
/**
 * feature functional component with parallax scroll
 */
const parallaxScroll = () => (
  <div className="parallex">
    <div className="parallex-menu">
      <div className="parallex-img">
        <span className="fa fa-dashboard fa-4x white-color" style={{ marginLeft: '40%' }} />
      </div>
      <div className="parallex-card">
        <div className="container"><br />
          <h2 className="white-color">Dashboard</h2>
          <p className="white-color">Enter your dashboard to perform awesome
                      functionality which include ordering for a meal, adding meal options
                      from meal menu
          </p>
        </div>
        <div className="parallex-footer">
          <button>
            <Link to="/dashbord"> more</Link>
          </button>
        </div>
      </div>
    </div>
    <div className="parallex-menu">
      <div className="parallex-img">
        <span className="fa fa-rocket fa-4x white-color" style={{ marginLeft: '40%' }} />
      </div>
      <div className="parallex-card">
        <div className="container"><br />
          <h2 className="white-color">Login/SignUp</h2>
          <p className="p-style">Login to order menu of your choice with just a single click</p>
        </div>
        <div className="parallex-footer">
          <button>
            <Link to="/login"> more</Link>
          </button>
        </div>
      </div>
    </div>
    <div className="parallex-menu">
      <div className="parallex-img">
        <span className="fa fa-user fa-4x white-color" style={{ marginLeft: '40%' }} />
      </div>
      <div className="parallex-card">
        <div className="container"><br />
          <h2 className="white-color">SignUp</h2>
          <p className="p-style">
            SignUp to get started. Login to order menu of your choice with just a
            single click
          </p>
        </div>
        <div className="parallex-footer">
          <button>
            <Link to="/register"> more</Link>
          </button>
        </div>
      </div>
    </div>
  </div>
);
export default parallaxScroll;
