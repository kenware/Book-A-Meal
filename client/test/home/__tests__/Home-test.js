import React from 'react';

import { Home } from '../../../components/home/index';
import renderer from 'react-test-renderer';


describe('Test Home Component', () => {
  const props = {
    loadMostOrderedMeal: jest.fn(),
    mostOrder: [
      {
        Meal: {
          name: 'rice',
          price: '400',
          description: 'good',
          image: 'image'
        }
      }
    ],
    mealFlip: [
      {
        Meal: {
          name: 'rice',
          price: '400',
          description: 'good',
          image: 'image'
        }
      }
    ]
  };
  it('renders correctly', () => {
    const tree = shallow(<Home {...props} />);
    expect(tree).toMatchSnapshot();
  });

  it('it should display one meal from mosOrdered meal props', () => {
    const tree = shallow(<Home {...props} />);
    expect(tree.find('.meal-menu-card').length).toEqual(1);
  });
  it('it should display one meal from mealCardFlip props', () => {
    const tree = shallow(<Home {...props} />);
    expect(tree.find('.meal-menu-flip').length).toEqual(1);
  });
  it('it should flip card on mouseEnter event', () => {
    const tree = shallow(<Home {...props} />);
    expect(tree).toMatchSnapshot();
    tree.find('#flip-card').simulate('mouseEnter');
    expect(tree).toMatchSnapshot();
  });
});
