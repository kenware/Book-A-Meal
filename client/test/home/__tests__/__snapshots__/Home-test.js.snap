// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home Component it should flip card on mouseEnter event 1`] = `
<div>
  <Connect(Nav)
    home="home"
  />
  <div
    className="carousel-header"
  >
    <div
      className="header-overlay"
    />
    <div
      className="caption"
    >
      <h3
        className="lead"
      >
        Book A Meal Today
      </h3>
      <p
        className="lead"
      >
        Health is wealth!
      </p>
    </div>
  </div>
  <div
    className="site-container"
    style={
      Object {
        "marginLeft": "3%",
        "marginRight": "3%",
      }
    }
  >
    <h2>
      BOOK A MEAL IN THREE EASY STEPS
    </h2>
    <div
      className="all-meal-step"
    >
      <div
        className="meal-day"
      >
        <div>
          <div
            className="circle"
          />
          <br />
          <span>
            SignUp/SignIn
          </span>
        </div>
      </div>
      <div
        className="meal-day"
      >
        <span
          className="fa fa-long-arrow-right fa-4x"
        />
      </div>
      <div
        className="meal-day"
      >
        <div>
          <div
            className="circle"
          />
          <br />
          <span>
            Order A Meal
          </span>
        </div>
      </div>
      <div
        className="meal-day"
      >
        <span
          className="fa fa-long-arrow-right fa-4x"
        />
      </div>
      <div
        className="meal-day"
      >
        <div>
          <div>
            <img
              alt="meal"
              className="rounded-circle"
              id="App-logo"
              src="image/l.png"
              style={
                Object {
                  "height": "100px",
                  "width": "100px",
                }
              }
            />
          </div>
          <br />
          <span>
            Hot Fresh Meal
          </span>
        </div>
      </div>
    </div>
    <h2>
      Recommended meals
    </h2>
    <div
      className="meal-menu-row"
    >
      <div
        className="meal-menu"
      >
        <div
          className="meal-menu-card"
        >
          <div
            className="meal-img-wrap"
          >
            <div
              className="meal-overlay"
            />
            <img
              alt="Avatar"
              className="meal-card-img"
              src="image"
            />
          </div>
          <div
            className="container l-r-pad-text"
          >
            <h4
              className="p-color"
            >
              <b>
                $
                400
              </b>
            </h4>
            <p>
              rice
            </p>
            <Link
              replace={false}
              to="/detail/undefined"
            >
              View
            </Link>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="parallex"
  >
    <div
      className="parallex-menu"
    >
      <div
        className="parallex-img"
      >
        <span
          className="fa fa-dashboard fa-4x white-color"
          style={
            Object {
              "marginLeft": "40%",
            }
          }
        />
      </div>
      <div
        className="parallex-card"
      >
        <div
          className="container"
        >
          <br />
          <h2
            className="white-color"
          >
            Dashboard
          </h2>
          <p
            className="white-color"
          >
            Enter your dashboard to perform awesome functionality which include ordering for a meal, adding meal options from meal menu
          </p>
        </div>
        <div
          className="parallex-footer"
        >
          <button>
            <Link
              replace={false}
              to="/dashbord"
            >
               more
            </Link>
          </button>
        </div>
      </div>
    </div>
    <div
      className="parallex-menu"
    >
      <div
        className="parallex-img"
      >
        <span
          className="fa fa-rocket fa-4x white-color"
          style={
            Object {
              "marginLeft": "40%",
            }
          }
        />
      </div>
      <div
        className="parallex-card"
      >
        <div
          className="container"
        >
          <br />
          <h2
            className="white-color"
          >
            Login/SignUp
          </h2>
          <p
            className="p-style"
          >
            Login to order menu of your choice with just a single click
          </p>
        </div>
        <div
          className="parallex-footer"
        >
          <button>
            <Link
              replace={false}
              to="/login"
            >
               more
            </Link>
          </button>
        </div>
      </div>
    </div>
    <div
      className="parallex-menu"
    >
      <div
        className="parallex-img"
      >
        <span
          className="fa fa-user fa-4x white-color"
          style={
            Object {
              "marginLeft": "40%",
            }
          }
        />
      </div>
      <div
        className="parallex-card"
      >
        <div
          className="container"
        >
          <br />
          <h2
            className="white-color"
          >
            SignUp
          </h2>
          <p
            className="p-style"
          >
            SignUp to get started. Login to order menu of your choice with just a single click
          </p>
        </div>
        <div
          className="parallex-footer"
        >
          <button>
            <Link
              replace={false}
              to="/register"
            >
               more
            </Link>
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="site-container"
    style={
      Object {
        "marginBottom": "2rem",
        "marginLeft": "3%",
        "marginRight": "3%",
      }
    }
  >
    <div
      className="meal-menu-row"
    >
      <div
        className="meal-menu"
      >
        <div
          className="meal-menu-flip"
        >
          <ReactCardFlip
            cardStyles={
              Object {
                "back": Object {},
                "front": Object {},
              }
            }
            flipSpeedBackToFront={0.6}
            flipSpeedFrontToBack={0.6}
            infinite={false}
            isFlipped={false}
          >
            <div
              id="flip-card"
              key="front"
              onMouseEnter={[Function]}
            >
              <img
                alt="Avatar"
                className="meal-card-img"
                src="image"
              />
            </div>
            <div
              key="back"
              onMouseLeave={[Function]}
            >
              <h2>
                rice
              </h2>
              <h2>
                Price: 
                400
              </h2>
              <p
                className="justify l-r-pad-text"
              >
                good...
              </p>
            </div>
          </ReactCardFlip>
        </div>
      </div>
    </div>
  </div>
  <div
    className="user-testimonial"
  >
    <br />
    <br />
    <h2>
      WHAT OUR FANTASTIC USERS SAY
    </h2>
    <h4
      className="p-color text-center"
    >
      Best site to get latest recipe with just a click
      <br />
      <br />
    </h4>
    <div
      className="testimonial-wrap"
    >
      <div>
        <div
          className="user-testimonial-menu"
        >
          <div
            className="testimonial-photo"
          >
            <img
              alt="profile"
              className="rounded-circle"
              src="image/eze.jpg"
              style={
                Object {
                  "height": "100px",
                  "width": "100px",
                }
              }
            />
          </div>
          <p
            className="justify text-pad"
          >
            Book-A-Meal is just amazing!. For the first time i can book for any meal by a single click any where around the world, without having to go to any  resturant .This is really awesome.
          </p>
          <div
            className="container"
          >
            <h2
              className="white-color"
            >
              Kevin
            </h2>
            <p
              className="text-center white-color"
            >
              Developer
            </p>
          </div>
        </div>
      </div>
      <div>
        <div
          className="user-testimonial-menu"
        >
          <div
            className="testimonial-photo"
          >
            <img
              alt="profile"
              className="rounded-circle"
              src="image/face1.jpg"
              style={
                Object {
                  "height": "100px",
                  "width": "100px",
                }
              }
            />
          </div>
          <p
            className="justify l-r-pad-text"
          >
            Book-A-Meal is  awesome!.This have preventd the stress of having to go down the street from one resturant to another, i have my favorite delivered to my door step b a single click.This is really awsome.
          </p>
          <div
            className="container"
          >
            <h2
              className="white-color"
            >
              Jessica Jones
            </h2>
            <p
              className="white-color text-center"
            >
              Actor
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <footer />
</div>
`;

exports[`Test Home Component it should flip card on mouseEnter event 2`] = `
<div>
  <Connect(Nav)
    home="home"
  />
  <div
    className="carousel-header"
  >
    <div
      className="header-overlay"
    />
    <div
      className="caption"
    >
      <h3
        className="lead"
      >
        Book A Meal Today
      </h3>
      <p
        className="lead"
      >
        Health is wealth!
      </p>
    </div>
  </div>
  <div
    className="site-container"
    style={
      Object {
        "marginLeft": "3%",
        "marginRight": "3%",
      }
    }
  >
    <h2>
      BOOK A MEAL IN THREE EASY STEPS
    </h2>
    <div
      className="all-meal-step"
    >
      <div
        className="meal-day"
      >
        <div>
          <div
            className="circle"
          />
          <br />
          <span>
            SignUp/SignIn
          </span>
        </div>
      </div>
      <div
        className="meal-day"
      >
        <span
          className="fa fa-long-arrow-right fa-4x"
        />
      </div>
      <div
        className="meal-day"
      >
        <div>
          <div
            className="circle"
          />
          <br />
          <span>
            Order A Meal
          </span>
        </div>
      </div>
      <div
        className="meal-day"
      >
        <span
          className="fa fa-long-arrow-right fa-4x"
        />
      </div>
      <div
        className="meal-day"
      >
        <div>
          <div>
            <img
              alt="meal"
              className="rounded-circle"
              id="App-logo"
              src="image/l.png"
              style={
                Object {
                  "height": "100px",
                  "width": "100px",
                }
              }
            />
          </div>
          <br />
          <span>
            Hot Fresh Meal
          </span>
        </div>
      </div>
    </div>
    <h2>
      Recommended meals
    </h2>
    <div
      className="meal-menu-row"
    >
      <div
        className="meal-menu"
      >
        <div
          className="meal-menu-card"
        >
          <div
            className="meal-img-wrap"
          >
            <div
              className="meal-overlay"
            />
            <img
              alt="Avatar"
              className="meal-card-img"
              src="image"
            />
          </div>
          <div
            className="container l-r-pad-text"
          >
            <h4
              className="p-color"
            >
              <b>
                $
                400
              </b>
            </h4>
            <p>
              rice
            </p>
            <Link
              replace={false}
              to="/detail/undefined"
            >
              View
            </Link>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="parallex"
  >
    <div
      className="parallex-menu"
    >
      <div
        className="parallex-img"
      >
        <span
          className="fa fa-dashboard fa-4x white-color"
          style={
            Object {
              "marginLeft": "40%",
            }
          }
        />
      </div>
      <div
        className="parallex-card"
      >
        <div
          className="container"
        >
          <br />
          <h2
            className="white-color"
          >
            Dashboard
          </h2>
          <p
            className="white-color"
          >
            Enter your dashboard to perform awesome functionality which include ordering for a meal, adding meal options from meal menu
          </p>
        </div>
        <div
          className="parallex-footer"
        >
          <button>
            <Link
              replace={false}
              to="/dashbord"
            >
               more
            </Link>
          </button>
        </div>
      </div>
    </div>
    <div
      className="parallex-menu"
    >
      <div
        className="parallex-img"
      >
        <span
          className="fa fa-rocket fa-4x white-color"
          style={
            Object {
              "marginLeft": "40%",
            }
          }
        />
      </div>
      <div
        className="parallex-card"
      >
        <div
          className="container"
        >
          <br />
          <h2
            className="white-color"
          >
            Login/SignUp
          </h2>
          <p
            className="p-style"
          >
            Login to order menu of your choice with just a single click
          </p>
        </div>
        <div
          className="parallex-footer"
        >
          <button>
            <Link
              replace={false}
              to="/login"
            >
               more
            </Link>
          </button>
        </div>
      </div>
    </div>
    <div
      className="parallex-menu"
    >
      <div
        className="parallex-img"
      >
        <span
          className="fa fa-user fa-4x white-color"
          style={
            Object {
              "marginLeft": "40%",
            }
          }
        />
      </div>
      <div
        className="parallex-card"
      >
        <div
          className="container"
        >
          <br />
          <h2
            className="white-color"
          >
            SignUp
          </h2>
          <p
            className="p-style"
          >
            SignUp to get started. Login to order menu of your choice with just a single click
          </p>
        </div>
        <div
          className="parallex-footer"
        >
          <button>
            <Link
              replace={false}
              to="/register"
            >
               more
            </Link>
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="site-container"
    style={
      Object {
        "marginBottom": "2rem",
        "marginLeft": "3%",
        "marginRight": "3%",
      }
    }
  >
    <div
      className="meal-menu-row"
    >
      <div
        className="meal-menu"
      >
        <div
          className="meal-menu-flip"
        >
          <ReactCardFlip
            cardStyles={
              Object {
                "back": Object {},
                "front": Object {},
              }
            }
            flipSpeedBackToFront={0.6}
            flipSpeedFrontToBack={0.6}
            infinite={false}
            isFlipped={true}
          >
            <div
              id="flip-card"
              key="front"
              onMouseEnter={[Function]}
            >
              <img
                alt="Avatar"
                className="meal-card-img"
                src="image"
              />
            </div>
            <div
              key="back"
              onMouseLeave={[Function]}
            >
              <h2>
                rice
              </h2>
              <h2>
                Price: 
                400
              </h2>
              <p
                className="justify l-r-pad-text"
              >
                good...
              </p>
            </div>
          </ReactCardFlip>
        </div>
      </div>
    </div>
  </div>
  <div
    className="user-testimonial"
  >
    <br />
    <br />
    <h2>
      WHAT OUR FANTASTIC USERS SAY
    </h2>
    <h4
      className="p-color text-center"
    >
      Best site to get latest recipe with just a click
      <br />
      <br />
    </h4>
    <div
      className="testimonial-wrap"
    >
      <div>
        <div
          className="user-testimonial-menu"
        >
          <div
            className="testimonial-photo"
          >
            <img
              alt="profile"
              className="rounded-circle"
              src="image/eze.jpg"
              style={
                Object {
                  "height": "100px",
                  "width": "100px",
                }
              }
            />
          </div>
          <p
            className="justify text-pad"
          >
            Book-A-Meal is just amazing!. For the first time i can book for any meal by a single click any where around the world, without having to go to any  resturant .This is really awesome.
          </p>
          <div
            className="container"
          >
            <h2
              className="white-color"
            >
              Kevin
            </h2>
            <p
              className="text-center white-color"
            >
              Developer
            </p>
          </div>
        </div>
      </div>
      <div>
        <div
          className="user-testimonial-menu"
        >
          <div
            className="testimonial-photo"
          >
            <img
              alt="profile"
              className="rounded-circle"
              src="image/face1.jpg"
              style={
                Object {
                  "height": "100px",
                  "width": "100px",
                }
              }
            />
          </div>
          <p
            className="justify l-r-pad-text"
          >
            Book-A-Meal is  awesome!.This have preventd the stress of having to go down the street from one resturant to another, i have my favorite delivered to my door step b a single click.This is really awsome.
          </p>
          <div
            className="container"
          >
            <h2
              className="white-color"
            >
              Jessica Jones
            </h2>
            <p
              className="white-color text-center"
            >
              Actor
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <footer />
</div>
`;

exports[`Test Home Component renders correctly 1`] = `
<div>
  <Connect(Nav)
    home="home"
  />
  <div
    className="carousel-header"
  >
    <div
      className="header-overlay"
    />
    <div
      className="caption"
    >
      <h3
        className="lead"
      >
        Book A Meal Today
      </h3>
      <p
        className="lead"
      >
        Health is wealth!
      </p>
    </div>
  </div>
  <div
    className="site-container"
    style={
      Object {
        "marginLeft": "3%",
        "marginRight": "3%",
      }
    }
  >
    <h2>
      BOOK A MEAL IN THREE EASY STEPS
    </h2>
    <div
      className="all-meal-step"
    >
      <div
        className="meal-day"
      >
        <div>
          <div
            className="circle"
          />
          <br />
          <span>
            SignUp/SignIn
          </span>
        </div>
      </div>
      <div
        className="meal-day"
      >
        <span
          className="fa fa-long-arrow-right fa-4x"
        />
      </div>
      <div
        className="meal-day"
      >
        <div>
          <div
            className="circle"
          />
          <br />
          <span>
            Order A Meal
          </span>
        </div>
      </div>
      <div
        className="meal-day"
      >
        <span
          className="fa fa-long-arrow-right fa-4x"
        />
      </div>
      <div
        className="meal-day"
      >
        <div>
          <div>
            <img
              alt="meal"
              className="rounded-circle"
              id="App-logo"
              src="image/l.png"
              style={
                Object {
                  "height": "100px",
                  "width": "100px",
                }
              }
            />
          </div>
          <br />
          <span>
            Hot Fresh Meal
          </span>
        </div>
      </div>
    </div>
    <h2>
      Recommended meals
    </h2>
    <div
      className="meal-menu-row"
    >
      <div
        className="meal-menu"
      >
        <div
          className="meal-menu-card"
        >
          <div
            className="meal-img-wrap"
          >
            <div
              className="meal-overlay"
            />
            <img
              alt="Avatar"
              className="meal-card-img"
              src="image"
            />
          </div>
          <div
            className="container l-r-pad-text"
          >
            <h4
              className="p-color"
            >
              <b>
                $
                400
              </b>
            </h4>
            <p>
              rice
            </p>
            <Link
              replace={false}
              to="/detail/undefined"
            >
              View
            </Link>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="parallex"
  >
    <div
      className="parallex-menu"
    >
      <div
        className="parallex-img"
      >
        <span
          className="fa fa-dashboard fa-4x white-color"
          style={
            Object {
              "marginLeft": "40%",
            }
          }
        />
      </div>
      <div
        className="parallex-card"
      >
        <div
          className="container"
        >
          <br />
          <h2
            className="white-color"
          >
            Dashboard
          </h2>
          <p
            className="white-color"
          >
            Enter your dashboard to perform awesome functionality which include ordering for a meal, adding meal options from meal menu
          </p>
        </div>
        <div
          className="parallex-footer"
        >
          <button>
            <Link
              replace={false}
              to="/dashbord"
            >
               more
            </Link>
          </button>
        </div>
      </div>
    </div>
    <div
      className="parallex-menu"
    >
      <div
        className="parallex-img"
      >
        <span
          className="fa fa-rocket fa-4x white-color"
          style={
            Object {
              "marginLeft": "40%",
            }
          }
        />
      </div>
      <div
        className="parallex-card"
      >
        <div
          className="container"
        >
          <br />
          <h2
            className="white-color"
          >
            Login/SignUp
          </h2>
          <p
            className="p-style"
          >
            Login to order menu of your choice with just a single click
          </p>
        </div>
        <div
          className="parallex-footer"
        >
          <button>
            <Link
              replace={false}
              to="/login"
            >
               more
            </Link>
          </button>
        </div>
      </div>
    </div>
    <div
      className="parallex-menu"
    >
      <div
        className="parallex-img"
      >
        <span
          className="fa fa-user fa-4x white-color"
          style={
            Object {
              "marginLeft": "40%",
            }
          }
        />
      </div>
      <div
        className="parallex-card"
      >
        <div
          className="container"
        >
          <br />
          <h2
            className="white-color"
          >
            SignUp
          </h2>
          <p
            className="p-style"
          >
            SignUp to get started. Login to order menu of your choice with just a single click
          </p>
        </div>
        <div
          className="parallex-footer"
        >
          <button>
            <Link
              replace={false}
              to="/register"
            >
               more
            </Link>
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="site-container"
    style={
      Object {
        "marginBottom": "2rem",
        "marginLeft": "3%",
        "marginRight": "3%",
      }
    }
  >
    <div
      className="meal-menu-row"
    >
      <div
        className="meal-menu"
      >
        <div
          className="meal-menu-flip"
        >
          <ReactCardFlip
            cardStyles={
              Object {
                "back": Object {},
                "front": Object {},
              }
            }
            flipSpeedBackToFront={0.6}
            flipSpeedFrontToBack={0.6}
            infinite={false}
            isFlipped={false}
          >
            <div
              id="flip-card"
              key="front"
              onMouseEnter={[Function]}
            >
              <img
                alt="Avatar"
                className="meal-card-img"
                src="image"
              />
            </div>
            <div
              key="back"
              onMouseLeave={[Function]}
            >
              <h2>
                rice
              </h2>
              <h2>
                Price: 
                400
              </h2>
              <p
                className="justify l-r-pad-text"
              >
                good...
              </p>
            </div>
          </ReactCardFlip>
        </div>
      </div>
    </div>
  </div>
  <div
    className="user-testimonial"
  >
    <br />
    <br />
    <h2>
      WHAT OUR FANTASTIC USERS SAY
    </h2>
    <h4
      className="p-color text-center"
    >
      Best site to get latest recipe with just a click
      <br />
      <br />
    </h4>
    <div
      className="testimonial-wrap"
    >
      <div>
        <div
          className="user-testimonial-menu"
        >
          <div
            className="testimonial-photo"
          >
            <img
              alt="profile"
              className="rounded-circle"
              src="image/eze.jpg"
              style={
                Object {
                  "height": "100px",
                  "width": "100px",
                }
              }
            />
          </div>
          <p
            className="justify text-pad"
          >
            Book-A-Meal is just amazing!. For the first time i can book for any meal by a single click any where around the world, without having to go to any  resturant .This is really awesome.
          </p>
          <div
            className="container"
          >
            <h2
              className="white-color"
            >
              Kevin
            </h2>
            <p
              className="text-center white-color"
            >
              Developer
            </p>
          </div>
        </div>
      </div>
      <div>
        <div
          className="user-testimonial-menu"
        >
          <div
            className="testimonial-photo"
          >
            <img
              alt="profile"
              className="rounded-circle"
              src="image/face1.jpg"
              style={
                Object {
                  "height": "100px",
                  "width": "100px",
                }
              }
            />
          </div>
          <p
            className="justify l-r-pad-text"
          >
            Book-A-Meal is  awesome!.This have preventd the stress of having to go down the street from one resturant to another, i have my favorite delivered to my door step b a single click.This is really awsome.
          </p>
          <div
            className="container"
          >
            <h2
              className="white-color"
            >
              Jessica Jones
            </h2>
            <p
              className="white-color text-center"
            >
              Actor
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <footer />
</div>
`;
